pokemongo_bot/inventory.py

```diff
@@ -108,7 +108,13 @@ class Items(_BaseInventoryComponent):
     STATIC_DATA_FILE = os.path.join(_base_dir, 'data', 'items.json')
 
     def count_for(self, item_id):
-        return self._data[item_id]['count']
+        try:
+            return self._data[item_id]['count']
+        except KeyError:
+            return 0
+
+    def decrement_count(self, item_id):
```

Comment:
Is it worth adding a test for this to prove that we can increment, decrement, and call count_for and get results that we expect?